VK/VK/Presentation/Authentification/ViewControllers/AuthentificationViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  AuthentificationViewController.swift
+//  VK
+//
+//  Created by Алена Панченко on 22.11.2022.
+//
+
+import UIKit
+
+import WebKit
+
+/// Экран аутентификации
+final class AuthentificationViewController: UIViewController {
+    
+    // MARK: - Private Constants
+    private enum Constants {
+        static let urlComponentsSchemeName = "https"
```

Comment:
```suggestion
        static let urlComponentsSchemeName = "https"
```
сейчас наверное можно не переделывать, но одинаковые константы хочется вынести в отдельный файл что бы использовать по всему проекту